app/Http/Controllers/FrontController.php

```diff
@@ -140,13 +140,13 @@ protected function getEditorialHeaderData()
     protected function getRelatedSidebarItemsData()
     {
         $itemId = request()->query('id');
-        $itemType = request()->query('model');
+        $itemModel = app(request()->query('model'));
 
-        if (app($itemType)->hasAugmentedModel()) {
-            $itemType = app($itemType)->getAugmentedModelClass();
-            $item = $itemType::where('datahub_id', $itemId)->first();
+        if ((method_exists($itemModel, 'hasAugmentedModel')) && $itemModel->hasAugmentedModel()) {
+            $itemModel = $itemModel->getAugmentedModelClass();
+            $item = $itemModel::where('datahub_id', $itemId)->first();
```

Comment:
I think you can use the `getAugmentedModel()` method here: https://github.com/art-institute-of-chicago/artic.edu/blob/develop/app/Libraries/Api/Models/Behaviors/HasAugmentedModel.php#L23

Reply:
@zachgarwood I need to build the local/augmented class not the Api one to query. This check here is to see if the initial model is an API one and then we query for the local data. Not the reverse like we typically do which is confusing lol